tools/please_go/BUILD

```diff
@@ -26,6 +26,7 @@ genrule(
     ],
     outs = ["please_go.bootstrap"],
     binary = True,
+    sandbox = False,
```

Comment:
Doesn't this make it unsandboxed?

Reply:
Yeah, it's building it with `go build` to bootstrap so it can't be sandboxed.

Reply:
Right yeah that was my confusion. The sandboxed version does work at present though? Is there some specific case it doesn't?

Reply:
It doesn't work if you try and build this from a repo with sandboxing enabled.